src/containerapp/HISTORY.rst

```diff
@@ -11,6 +11,7 @@ upcoming
 * 'az containerapp env telemetry otlp remove': Support remove environment otlp configuration with --otlp-name
 * 'az containerapp env telemetry otlp show': Support show environment otlp configuration with --otlp-name
 * 'az containerapp env telemetry otlp list': Support show environment otlp configurations
+* Wait long time for logstream of Cloud Build to the 'az containerapp up'/'az containerapp create' and 'az containerapp update' commands
```

Comment:
Please add what is the commands related to this change.

Reply:
For example:
'az containerapp up/create/update': Wait longer time for logstream of Cloud Build to make sure the container start

Reply:
good, fixed